samples/todolist-aspnetcore/Startup.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OpenApi.Models;
+using TodoApi.Models;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace TodoApi
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.AddDurableTask();
```

Comment:
Consider adding a longer comment and moving this up to the top of this method so that it highlights that this is the key to register the IDurableClientFactory as a service so you can consume it.